tutorials/notebooks/keras/ptq/keras_deeplabv3plus_for_imx500.ipynb

```diff
@@ -0,0 +1,342 @@
+    "\n",
+    "Now, we're all set to use MCT's post-training quantization. To begin, we'll define a representative dataset and proceed with the model quantization. Please note that, for demonstration purposes, we'll use the evaluation dataset as our representative dataset. We'll calibrate the model using 320 images, divided into 20 iterations of 'batch_size' images each. "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "outputs": [],
+   "source": [
+    "import model_compression_toolkit as mct\n",
+    "from typing import Iterator, List\n",
+    "\n",
+    "\n",
+    "n_iters = 20\n",
+    "\n",
+    "\n",
+    "# Define representative dataset generator\n",
+    "def get_representative_dataset():\n",
+    "    \"\"\"\n",
+    "    This function creates a representative dataset generator. The generator yields numpy\n",
+    "    arrays of batches of shape: [Batch, H, W ,C].\n",
+    "    \n",
+    "    Returns:\n",
+    "        A representative dataset generator\n",
+    "    \"\"\"       \n",
+    "\n",
+    "    representative_dataset = PascalDataset()\n",
+    "    representative_dataset.shuffle()\n",
+    "    \n",
+    "    \n",
+    "    def _representative_dataset() -> Iterator[List]:\n",
+    "        ds_iter = iter(representative_dataset)\n",
+    "        for _ in range(n_iters):\n",
+    "            yield [next(ds_iter)[0]]\n",
+    "\n",
+    "    return _representative_dataset\n",
+    "\n",
+    "\n",
+    "# Set IMX500-v1 TPC\n",
+    "tpc = mct.get_target_platform_capabilities(\"tensorflow\", 'imx500', target_platform_version='v1')\n",
+    "\n",
+    "# Perform post training quantization\n",
+    "quant_model, _ = mct.ptq.keras_post_training_quantization(model,\n",
+    "                                                          get_representative_dataset(),\n",
+    "                                                          target_platform_capabilities=tpc)\n",
+    "print('Quantized model is ready')"
+   ],
+   "metadata": {
+    "collapsed": false
+   },
+   "id": "56393342-cecf-4f64-b9ca-2f515c765942"
+  },
+  {
+   "cell_type": "markdown",
+   "source": [
+    "### Model Export\n",
+    "\n",
+    "Now, we can export the quantized model, ready for deployment, into a `.keras` format file. Please ensure that the `save_model_path` has been set correctly. "
+   ],
+   "metadata": {
+    "collapsed": false
+   },
+   "id": "3be2016acdc9da60"
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "outputs": [],
+   "source": [
+    "mct.exporter.keras_export_model(model=quant_model, save_model_path='qmodel.keras')"
+   ],
+   "metadata": {
+    "collapsed": false
+   },
+   "id": "72dd885c7b92fa93"
+  },
+  {
+   "cell_type": "markdown",
+   "id": "015e760b-6555-45b4-aaf9-500e974c1d86",
+   "metadata": {},
+   "source": [
+    "## Evaluation on Pascal dataset\n",
+    "\n",
+    "### Floating point model evaluation"
+   ]
+  },
+  {
+   "cell_type": "code",
```

Comment:
A few words before the code... in this step we evaluate the floating point model on PascalVoc using tensorflow MeanIoU metric.., etc.